lib/knapsack_pro/test_case_detectors/rspec_test_example_detector.rb

```diff
@@ -33,32 +33,22 @@ def generate_json_report(rspec_args)
         # Apply a --format option which overrides formatters from the RSpec custom option files like `.rspec`.
         cli_args = cli_args_without_formatters + cli_format + [
           '--dry-run',
-          '--no-color',
           '--out', report_path,
-          '--default-path', test_dir,
+          '--default-path', test_dir
         ] + KnapsackPro::TestFilePresenter.paths(test_file_entities)
-        options = ::RSpec::Core::ConfigurationOptions.new(cli_args)
-        exit_code = ::RSpec::Core::Runner.new(options).run($stderr, $stdout)
-        if exit_code != 0
-          debug_cmd = ([
-            'bundle exec rspec',
-          ] + cli_args).join(' ')
-
-          KnapsackPro.logger.error('-'*10 + ' START of actionable error message ' + '-'*50)
-          KnapsackPro.logger.error('RSpec (with a dry-run option) had a problem generating the report with test examples for the slow test files. Here is what you can do:')
-
-          KnapsackPro.logger.error("a) Please look for an error message from RSpec in the output above or below. If you don't see anything, that is fine. Sometimes RSpec does not produce any errors in the output.")
-
-          KnapsackPro.logger.error("b) Check if RSpec generated the report file #{report_path}. If the report exists, it may contain an error message. Here is a preview of the report file:")
-          KnapsackPro.logger.error(report_content || 'N/A')
-
-          KnapsackPro.logger.error('c) To reproduce the error manually, please run the following RSpec command. This way, you can find out what is causing the error. Please ensure you run the command in the same environment where the error occurred. For instance, if the error happens on the CI server, you should run the command in the CI environment:')
-          KnapsackPro.logger.error(debug_cmd)
+        exit_code = begin
+          options = ::RSpec::Core::ConfigurationOptions.new(cli_args)
+          ::RSpec::Core::Runner.new(options).run($stderr, $stdout)
+        rescue SystemExit => e
+          e.status
+        end
 
-          KnapsackPro.logger.error('-'*10 + ' END of actionable error message ' + '-'*50)
+        return if exit_code.zero?
 
-          raise 'There was a problem while generating test examples for the slow test files. Please read the actionable error message above.'
-        end
+        report.fetch('messages', []).each { |message| puts message }
+        command = (['bundle exec rspec'] + cli_args).join(' ')
+        KnapsackPro.logger.error("Failed to generate the slow test files report: #{command}")
```

Comment:
suggestion

```suggestion
        KnapsackPro.logger.error("Failed to generate the slow test files report. To reproduce the issue, run the following command locally: #{command}")
```

Reply:
I think it's self-explanatory. I'm leaving as is.